src/components/BookList.js

```diff
@@ -0,0 +1,29 @@
+/* eslint-disable react/jsx-key */
```

Comment:
- [ ] Since `linter` are there to help us stick to best practices, it will be nice to fix the issue instead of disabling `eslint`. Please kindly fix the issue.

Reply:
done